src/lang-swift.js

```diff
@@ -0,0 +1,59 @@
+// Copyright (C) 2015 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview
+ * Registers a language handler for Swift
+ *
+ *
+ * To use, include prettify.js and this file in your HTML page.
+ * Then put your code in an HTML tag like
+ *      <pre class="prettyprint lang-swift">(my swift code)</pre>
+ * This file supports the following language extensions:
+ *     lang-swift - Swift
+ *
+ * I used https://developer.apple.com/library/ios/documentation/Swift/Conceptual/Swift_Programming_Language/AboutTheLanguageReference.html
+ * as the basis for this. In particular, I targeted the revision from
+ * 2015-04-08. Swift is still evolving, and this was the latest version 
+ * available at the time. I will keep the code unoptimized to ease changes that come with new Swift standards.
+ *
+ * @author cerech@google.com
+ */
+
+PR['registerLangHandler'](
+    PR['createSimpleLexer'](
+        [
+          //whitespace
+          [PR['PR_PLAIN'],                /^[ \n\r\t\v\f\0]+/, null, ' \n\r\t\v\f\0'],
+          //string literals
+          [PR['PR_STRING'],               /^".*?"/, null, '"'],
```

Comment:
Is `"\"` a complete string literal?

[Lexical Structure](https://developer.apple.com/library/prerelease/ios/documentation/Swift/Conceptual/Swift_Programming_Language/LexicalStructure.html#//apple_ref/doc/uid/TP40014097-CH30-ID410)  seems to suggest that

`"\""` is a full lexical token and `"\(""\)"` is also a full token.

Perhaps
`/^(?:"|\\[)])(?:[^\\\"]|\\[^(])*(?:"|\\[(])/`